### PR TITLE
Azure Flexible DB for MySQL Document

### PR DESCRIPTION
**Pull request by sachin-sumologic**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@sachin-sumologic, I made some minor changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4665/commits/321f848b6d1227a722a42f00a68408cfce74dd5c)").

The dashboards sections need a little more work:
- The screenshots should be cropped to remove extra space
- Each dashboard needs a description. For examples, see:
   - [Azure Application Gateway dashboards](https://help.sumologic.com/docs/integrations/microsoft-azure/azure-application-gateway/#viewing-the-azure-application-gateway-dashboards)
   - [Azure Functions](https://help.sumologic.com/docs/integrations/microsoft-azure/azure-functions/)

File: docs/integrations/microsoft-azure/azure-database-for-mysql.md [hunk 1]
```diff
@@ -16,46 +16,226 @@ The below instructions applies to Azure Database for MySQL with Flexible Server
...
+* **MySQL Error Logs**. In Azure Database for MySQL flexible server, the error log is available to users to configure and access. Error logs in MySQL gather diagnostic messages during server startup and shutdown and while the server is running, to provide information that can help proactive troubleshooting. For more information about the MySQL error log, see the [Error Log Documentation](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-error-logs).
+   :::note
+   MySQL Error Logs are under in a preview phase. Error logs are available under Server Logs only, and error logs can't be emitted to Azure diagnostic logs. You need to download logs from the **Server Logs > Error Logs** menu send them to a Sumo Logic HTTP collector endpoint via curl request.
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-database-for-mysql.md:23]:
```suggestion
   MySQL error logs are currently in a preview phase and are only available under Server Logs. These logs cannot be emitted to Azure diagnostic logs directly. To access the error logs, navigate to **Server Logs > Error Logs**, download them, and then send the logs to a Sumo Logic HTTP collector endpoint using a `curl` request.
```

File: docs/integrations/microsoft-azure/azure-database-for-mysql.md [hunk 2]
```diff
@@ -16,46 +16,226 @@ The below instructions applies to Azure Database for MySQL with Flexible Server
...
+1. To create the diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the steps below for each Azure Redis cache account that you want to monitor.
+   1. Choose **Stream to an event hub** as the destination.
+   1. Select `allLogs`.
+   1. Use the Event Hub namespace and Event Hub name configured in the previous step in the destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+1. Tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Redis Cache Tag Location" style={{border: '1px solid gray'}} width="400" /> <br/><img src={useBaseUrl('img/send-data/azureflexible-mysqlserver-logs.png')} alt="Azure flexible mysql server logs" style={{border: '1px solid gray'}} width="800" /> 
+
+#### Activity Logs
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Do not perform this step if you are already collecting activity logs for a subscription.
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-database-for-mysql.md:147]:
```suggestion
To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Skip this step if you are already collecting activity logs for a subscription.
```

File: docs/integrations/microsoft-azure/azure-database-for-mysql.md [hunk 3]
```diff
@@ -16,46 +16,226 @@ The below instructions applies to Azure Database for MySQL with Flexible Server
...
+
+## Viewing the  Flexible Database for Mysql dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Error Logs
+
+The **Azure Database for Mysql - Error Logs**
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-database-for-mysql.md:169]: I'd eliminate the `The` in these sections since it's not a full sentence.

**Review by kimsauce — APPROVED** [review]
had a couple suggestions but looks great

File: docs/integrations/microsoft-azure/azure-database-for-mysql.md (hunk 3 above)
Comment (JV0812) [reply docs/integrations/microsoft-azure/azure-database-for-mysql.md:169]: We need to add more information to the dashboard description. cc: @sachin-sumologic

**Review by jpipkin1 — APPROVED** [review]
Looks great, @sachin-sumologic. I made minor changes (removed "etc" per commit "[Minor updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4665/commits/0bef0a479d750709442c398bd78c4588cbb52a52)").

File: docs/integrations/microsoft-azure/azure-database-for-mysql.md [hunk 4]
```diff
@@ -16,46 +16,226 @@ The below instructions applies to Azure Database for MySQL with Flexible Server
...
+* **MySQL Error Logs**. In Azure Database for MySQL flexible server, the error log is available to users to configure and access. Error logs in MySQL gather diagnostic messages during server startup and shutdown and while the server is running, to provide information that can help proactive troubleshooting. For more information about the MySQL error log, see the [Error Log Documentation](https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-error-logs).
+   :::note
+   MySQL Error Logs are currently in a preview phase and are only available under Server Logs. These logs cannot be emitted to Azure diagnostic logs directly. To access the error logs, navigate to **Server Logs > Error Logs**, download them, and then send the logs to a Sumo Logic HTTP collector endpoint using a `curl` request.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-database-for-mysql.md:23]:
```suggestion
then [upload the logs](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/upload-logs) to a Sumo Logic HTTP source endpoint.
```

File: docs/integrations/microsoft-azure/azure-database-for-mysql.md [hunk 5]
```diff
@@ -16,46 +16,226 @@ The below instructions applies to Azure Database for MySQL with Flexible Server
...
+1. To set up the Azure Event Hubs source in Sumo Logic, refer to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+1. To create the diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the steps below for each Azure Redis cache account that you want to monitor.
+   1. Choose **Stream to an event hub** as the destination.
+   1. Select `allLogs`.
+   1. Use the Event Hub namespace and Event Hub name configured in the previous step in the destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+1. Tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Redis Cache Tag Location" style={{border: '1px solid gray'}} width="400" /> <br/><img src={useBaseUrl('img/send-data/azureflexible-mysqlserver-logs.png')} alt="Azure flexible mysql server logs" style={{border: '1px solid gray'}} width="800" /> 
+
+#### Activity Logs
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-database-for-mysql.md:145]: Need to add section for 
Enabling error logs (along with the required query parameters)
Enabling Slow query logs (along with the required query parameters)
Enabling Audit  logs (along with the required query parameters)


Also is there any extra step required for enabling any health related logs and security (cloud defender ) related logs?

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**